webhook_to_fedora_messaging_messages/base.py

```diff
@@ -23,7 +22,7 @@ def app_icon(self):
     @property
     def agent_name(self):
         """The username of the user who initiated the action that generated this message."""
-        return self.body.get("agent")
+        return self.body['body'].get("agent")
```

Comment:
If I understand correctly, "`body`" will contain the webhook body unmodified. If the app adds the `agent` key, it'll be at the root of the message body, so `self.body.get("agent")`.

Reply:
The body is a dictionary contains both message body and headers. Like GithubMessageV1(body={body: dict, headers: dict})